defi/proof-of-reserves/cli/test.ts

```diff
@@ -0,0 +1,58 @@
+import { IPoRAdapter } from '../types';
+import fs from 'fs';
+
+// usage
+// ts-node cli/test.ts project-slug
+// ex: ts-node cli/test.ts lombard
+
+const projectArgv = process.argv[2];
+
+const projects: Array<string> = [];
+if (projectArgv === 'allProtocols') {
+  const items = fs.readdirSync(`${__dirname}/../adapters`);
+  for (const item of items) {
+    let adapterName = item;
+    if (item.includes('.ts')) {
+      adapterName = item.split('.ts')[0];
+    }
+
+    projects.push(adapterName);
+  }
+} else {
+  projects.push(projectArgv);
+}
+
+(async function (){
+  for (const project of projects) {
+    let adapter: IPoRAdapter | null = null;
+  
+    try {
+      const adapterFile = `../adapters/${project}`;
+      adapter = (await import(adapterFile)).default;
+    } catch(e: any) {
+      console.log(`adapter ${project} not found`);
+      process.exit(0);
```

Comment:
no, failing silently is a bad idea. also, we should not stop because single adapter is broken.


also, can you store each result run, and at the end of all the runs, we show a table with data, and if `DISCORD_WEBHOOK` env field is set, you send error info like here? https://github.com/DefiLlama/defillama-server/blob/master/defi/src/api2/scripts/checkForProtocolsMissingTokenInfo.ts#L273